assets/app/features/upgrade/upgrade.controller.js

```diff
@@ -47,5 +49,14 @@
         function isLastStep() {
             return vm.steps.list[vm.steps.list.length - 1] === upgradeStepsFactory.activeStep;
         }
+
+        /**
+         * Trigger cancellation of the upgrade process and go back to landing page
+         */
+        function cancelUpgrade() {
+            upgradeFactory.cancelUpgrade().then(function(/* response */) {
+                $state.go('upgrade-landing');
```

Comment:
After cancelation, user needs to be redirected to crowbar dashboard.
Completely out of the scope of this PR, but I'd like to ask you to add a comment in the code and https://trello.com/c/aP3CKeqb/28-2-17-cancellation-routine-page-creation to not lose track of it